website/docs/quickstart/flink.md

```diff
@@ -509,3 +509,31 @@ to stop all containers.
 
 ## Learn more
 Now that you're up and running with Fluss and Flink, check out the [Apache Flink Engine](engine-flink/getting-started.md) docs to learn more features with Flink or [this guide](/maintenance/observability/quickstart.md) to learn how to set up an observability stack for Fluss and Flink.
+
```

Comment:
I think we could add this content to the `flink-cdc.md` document in the `engine-flink` folder.